components/DownloadButton.tsx

```diff
@@ -0,0 +1,79 @@
+import React, { useState, useEffect } from 'react'
+import { AntDesign as Icon } from '@expo/vector-icons'
+import * as FileSystem from 'expo-file-system'
+
+import { useMeditation, useAppDispatch } from '../hooks'
+import { filepaths } from '../redux/meditationSlice'
+import { store } from '../redux/store'
+import { useFiles } from '../hooks/useFiles'
+
+export default function DownloadButton(props: any) {
```

Comment:
Let's type the props properly here :) 

```
interface Props {
   id: string
}
```

```suggestion
export default function DownloadButton(props: Props) {
```